DEVX-1645:  Naming stateful operations (#314)
confluentinc/kafka-tutorials: _includes/tutorials/naming-changelog-repartition-topics/kstreams/markup/dev/add-filter-to-named-topology.adoc

Now let's add a new operator (`filter()`) to the named topology:

+++++
<pre class="snippet"><code class="java">
  KStream&lt;Long, Example&gt; inputStream = builder.stream(inputTopic, Consumed.with(longSerde, exampleSerde))
                                                  .selectKey((k, v) -> Long.parseLong(v.getName().substring(0, 1)))
                                                  .filter((k, v) -> k != 1L);
</code></pre>
+++++

But this time, adding a new processor won't change the name of the stateful parts of your application, as you've explicity named them in the previous step.

> Now go back to the window running the streams application and do a `CTRL+C` then restart the streams application with this command:

Reviewer: ```suggestion
In the terminal running the streams application, do a `CTRL+C`, then restart the streams application with this command:
```
